packages/utils/src/extractUrlsFromText.ts

```diff
@@ -0,0 +1,35 @@
+const urlRegex =
```

Comment:
nitpick: maybe it will be better to have it like this:
```suggestion
const URL_REGEX =
```

Reply:
done